quic/s2n-quic-platform/src/io/tokio/clock.rs

```diff
@@ -0,0 +1,98 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+use core::{
+    future::Future,
+    pin::Pin,
+    task::{Context, Poll},
+    time::Duration,
+};
+use s2n_quic_core::time::{self, Clock as ClockTrait, Timestamp};
+use tokio::time::{sleep_until, Instant, Sleep};
+
+const DEFAULT_TIMEOUT: Duration = Duration::from_secs(1);
```

Comment:
> /// Even if there is no progress to be made, wake up the task at least once a second

can you copy over this comment

Reply:
Like mentioned, this isn't true anymore. But I can add a comment as to why this exits.